{"ygPerms":{"resourceCapabilityList":[{"resourceType":"GROUP","capabilities":[{"name":"READ"},{"name":"JOIN"}]},{"resourceType":"PHOTO","capabilities":[]},{"resourceType":"FILE","capabilities":[]},{"resourceType":"MEMBER","capabilities":[]},{"resourceType":"LINK","capabilities":[]},{"resourceType":"CALENDAR","capabilities":[]},{"resourceType":"DATABASE","capabilities":[]},{"resourceType":"POLL","capabilities":[]},{"resourceType":"MESSAGE","capabilities":[{"name":"READ"}]},{"resourceType":"PENDING_MESSAGE","capabilities":[]},{"resourceType":"ATTACHMENTS","capabilities":[{"name":"READ"}]},{"resourceType":"PHOTOMATIC_ALBUMS","capabilities":[]},{"resourceType":"MEMBERSHIP_TYPE","capabilities":[]},{"resourceType":"POST","capabilities":[{"name":"READ"}]},{"resourceType":"PIN","capabilities":[]}],"groupUrl":"groups.yahoo.com","intlCode":"us"},"comscore":"pageview_candidate","ygData":{"userId":44728387,"authorName":"David Goodwin","from":"David Goodwin &lt;dgoodwin@...&gt;","profile":"dggoodwin","replyTo":"LIST","senderId":"lmJe-KEFnwcTfo2aiwhml7RmlDAEJ-CIGzfhH9uSgy2zaO3Poor0xhylZyoQHa4tXdxlICn5y-fXajS5uzCQ1kOp7VJqOkBuFC2u","spamInfo":{"isSpam":false,"reason":"0"},"subject":"Re: [cantera] ct2ctml error","postDate":"1100835480","msgId":589,"canDelete":false,"contentTrasformed":false,"systemMessage":false,"headers":{"messageIdInHeader":"PDY5NkQyMUFBLTM5REMtMTFEOS05NkVCLTAwMEQ5M0FFMjQxQUBjYWx0ZWNoLmVkdT4=","inReplyToHeader":"PGNuamphaytodWh0QGVHcm91cHMuY29tPg==","referencesHeader":"PGNuamphaytodWh0QGVHcm91cHMuY29tPg=="},"prevInTopic":588,"nextInTopic":590,"prevInTime":588,"nextInTime":590,"topicId":588,"numMessagesInTopic":7,"msgSnippet":"Tom - I m not sure, but if I were to guess, I would guess that Python is not finding the temporary Python script Cantera generates to process CTI input files.","rawEmail":"Return-Path: &lt;dgoodwin@...&gt;\r\nX-Sender: dgoodwin@...\r\nX-Apparently-To: cantera@yahoogroups.com\r\nReceived: (qmail 83667 invoked from network); 19 Nov 2004 03:38:04 -0000\r\nReceived: from unknown (66.218.66.166)\n  by m23.grp.scd.yahoo.com with QMQP; 19 Nov 2004 03:38:04 -0000\r\nReceived: from unknown (HELO outgoing-mail.its.caltech.edu) (131.215.49.69)\n  by mta5.grp.scd.yahoo.com with SMTP; 19 Nov 2004 03:38:04 -0000\r\nReceived: from water-dog (water-dog [192.168.1.26])\n\tby earth-ox-postvirus (Postfix) with ESMTP id 323FA109D02\n\tfor &lt;cantera@yahoogroups.com&gt;; Thu, 18 Nov 2004 19:38:04 -0800 (PST)\r\nReceived: from water-ox ([192.168.1.10])\n\tby water-dog (MailMonitor for SMTP v1.2.2 ) ;\n\tThu, 18 Nov 2004 19:38:03 -0800 (PST)\r\nReceived: from [10.0.1.2] (charter-241-015.caltech.edu [131.215.241.15])\n\tby water-ox.its.caltech.edu (Postfix) with ESMTP id CEF3926AE93\n\tfor &lt;cantera@yahoogroups.com&gt;; Thu, 18 Nov 2004 19:38:02 -0800 (PST)\r\nMime-Version: 1.0 (Apple Message framework v619)\r\nIn-Reply-To: &lt;cnjjak+huht@...&gt;\r\nReferences: &lt;cnjjak+huht@...&gt;\r\nContent-Type: text/plain; charset=US-ASCII; format=flowed\r\nMessage-Id: &lt;696D21AA-39DC-11D9-96EB-000D93AE241A@...&gt;\r\nContent-Transfer-Encoding: 7bit\r\nDate: Thu, 18 Nov 2004 19:38:00 -0800\r\nTo: cantera@yahoogroups.com\r\nX-Mailer: Apple Mail (2.619)\r\nX-Spam-Status: No, hits=0.0 tagged_above=-100000.0 required=5.0 \r\nX-Spam-Level: \r\nX-eGroups-Remote-IP: 131.215.49.69\r\nFrom: David Goodwin &lt;dgoodwin@...&gt;\r\nSubject: Re: [cantera] ct2ctml error\r\nX-Yahoo-Group-Post: member; u=44728387\r\nX-Yahoo-Profile: dggoodwin\r\n\r\nTom - I&#39;m not sure, but if I were to guess, I would guess that Python \nis not finding the temporary Python script Cantera generates to process \nCTI input files.  Judging from the pathname that appears in the error \nmessage &quot;C:&#92;DOCUME~1&#92;tps&#92;LOCALS~1&#92;Temp/&quot; it looks like you are running \nthe demos from a directory with a long name, probably containing \nspaces.  Try copying the demos to a directory with a name that \nWindows/DOS does not mangle, or at least not as badly (short with no \nspaces). If that doesn&#39;t work, try running the temporary Python script \nmanually, and look for error messages - for example, type at a command \nprompt:\n\npython .check1100822275.pyw\n\nHope this helps.\n\nDave\n\n\n\nOn Nov 18, 2004, at 5:47 PM, -Tom wrote:\n\n&gt;\n&gt;\n&gt; Group:\n&gt;\n&gt; I have attempted to install Cantera 1.5.4 (ver win32-py2.3) onto a\n&gt; WinXP Pro SP2 machine.  I also am using Matlab 7.0.1 and Python 2.3.4\n&gt; w/Numeric-23.6.\n&gt;\n&gt; At this point, I am receiving &quot;Cantera Error!&quot; when I attempt to run\n&gt; any of the demos.  I followed the Win installation directions and have\n&gt; attempting several &quot;fixes&quot; to no success already.\n&gt;\n&gt; The error appears originate in the ct2ctml procedure.  The error\n&gt; indicators from both programs are listed below.\n&gt;\n&gt; Can anyone please offer any suggestion on how to address this error?\n&gt;\n&gt; Thank you,\n&gt; -tom\n&gt;\n&gt;\n&gt; From Matlab---------Start Error---------------------\n&gt;\n&gt; The system cannot find the path specified.\n&gt; cmd /C c:/python23/python\n&gt; C:&#92;DOCUME~1&#92;tps&#92;LOCALS~1&#92;Temp/.check1100822275.pyw&gt;&gt; log 2&gt;&1\n&gt; ##################################################\n&gt; ######################\n&gt; The Cantera Python interface is required in order to process\n&gt; Cantera input files, but it does not seem to be correctly installed.\n&gt; Check that you can invoke the Python interpreter with\n&gt; the command &quot;python&quot;, and that typing &quot;from Cantera import *&quot;\n&gt; at the Python prompt does not produce an error. If Python on your\n&gt; system\n&gt; is invoked with some other command, set environment variable\n&gt; PYTHON_CMD\n&gt; to the full path to the Python interpreter.\n&gt; ##################################################\n&gt; #######################\n&gt; ???\n&gt;\n&gt; ************************************************\n&gt;                 Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: ct2ctml\n&gt; Error:   could not convert input file to CTML\n&gt;  command line was: c:/python23/python\n&gt; C:&#92;DOCUME~1&#92;tps&#92;LOCALS~1&#92;Temp/.cttmp1100822275.pyw&gt; ct2ctml.log 2&gt;&1\n&gt; Error in Python installation\n&gt;\n&gt; Error in ==&gt; XML_Node.XML_Node at 11\n&gt;    x.id = ctmethods(10,15,0,src); % newxml(name)\n&gt;\n&gt; Error in ==&gt; Solution.Solution at 30\n&gt; doc = XML_Node(&#39;doc&#39;,src);\n&gt;\n&gt; Error in ==&gt; IdealGasMix at 39\n&gt;        s = Solution(a);\n&gt;\n&gt; Error in ==&gt; equil at 12\n&gt;    gas = IdealGasMix(&#39;gri30.cti&#39;);\n&gt;\n&gt; From Matlab:---------End Error---------------------\n&gt;\n&gt;\n&gt; From Python:------------Start Error--------------------\n&gt;&gt;&gt;&gt;\n&gt;  The system cannot find the path specified.\n&gt;\n&gt; cmd /C c:/python23/python C:&#92;DOCUME~1\n&gt; ps&#92;LOCALS~1&#92;Temp/.check1100822454.pyw&gt;&gt; log 2&gt;&1\n&gt;\n&gt; ##################################################\n&gt; ######################\n&gt;\n&gt; The Cantera Python interface is required in order to process\n&gt; Cantera input files, but it does not seem to be correctly installed.\n&gt;\n&gt; Check that you can invoke the Python interpreter with\n&gt; the command &quot;python&quot;, and that typing &quot;from Cantera import *&quot;\n&gt; at the Python prompt does not produce an error. If Python on your\n&gt; system\n&gt; is invoked with some other command, set environment variable\n&gt; PYTHON_CMD\n&gt; to the full path to the Python interpreter.\n&gt;\n&gt; ##################################################\n&gt; #######################\n&gt;\n&gt;\n&gt; Traceback (most recent call last):\n&gt;   File &quot;C:&#92;programs&#92;Cantera154&#92;demos&#92;Python&#92;mix1.py&quot;, line 25, in\n&gt; -toplevel-\n&gt;     gas_a = Air()\n&gt;   File &quot;C:&#92;programs&#92;Python23&#92;Lib&#92;site-packages&#92;Cantera&#92;gases.py&quot;, line\n&gt; 40, in Air\n&gt;     return Solution(src=&quot;air.cti&quot;, id=&quot;air&quot;)\n&gt;   File &quot;C:&#92;programs&#92;Python23&#92;Lib&#92;site-packages&#92;Cantera&#92;solution.py&quot;,\n&gt; line 38, in __init__\n&gt;     root = XML.XML_Node(name = &#39;doc&#39;, src = src, preprocess = 1)\n&gt;   File &quot;C:&#92;programs&#92;Python23&#92;Lib&#92;site-packages&#92;Cantera&#92;XML.py&quot;, line\n&gt; 35, in __init__\n&gt;     self._xml_id = _cantera.xml_get_XML_File(src)\n&gt; error:\n&gt;\n&gt; ************************************************\n&gt;                 Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: ct2ctml\n&gt; Error:   could not convert input file to CTML\n&gt;  command line was: c:/python23/python\n&gt; C:&#92;DOCUME~1&#92;tps&#92;LOCALS~1&#92;Temp/.cttmp1100822454.pyw&gt; ct2ctml.log 2&gt;&1\n&gt; Error in Python installation\n&gt;&gt;&gt;&gt;\n&gt;\n&gt; From Python:------------------End Error------------\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt; Yahoo! Groups Links\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n&gt;\n\n\n\n"}}